web/templates/attendance/index_coordinator.html.eex

```diff
@@ -32,8 +32,18 @@
             >
               <i class="material-icons">more_vert</i>
             </button>
-            <ul class="mdl-menu mdl-menu--bottom-right mdl-js-menu"
-                for="tr_menu_<%= offered_course.id %>"
+            <ul
+              class="
+                mdl-menu mdl-js-menu
+                <%=
+                  if index > 10 && index > length(@offered_courses)-4 do
```

Comment:
the problem was that the lowest list items had menus that went down, but they did not always have enough space, and they messed up with the margins and scroll position. that won't happen if there are not too much items (e.g. won't cover the full screen), that's why the 10 condition is there. the another condition is there because only the last 3 list item would have menus that goes up, because the other menus would have enough space